### DISCUSSION TITLE
Why is the default config hidden?

### DISCUSSION THREAD

**Discussion by Rem0ld** [LazyVim/LazyVim#129] [General]
Hello, thanks for the work for starters!
I'm starting a new configuration with your repo and there is something I find not user friendly.
The fact that default config is somewhere else than nvim.
I'm talking about options, keymaps, commands and all preconfigured plugins.
So if I want to change or use something in there I need to refer to documentation first.
Another point for keymaps, I basically pasted default config from documentation and if I want to disable I use <nop> which makes a bigger keymap file and the file become less readable. 
I'd rather have everything in the files so we can actually know what's happening and we could more easily customise if needed.
What's your thought on that?

**Comment by folke** [comment]
That's not how LazyVim works. You import the LazyVim specs and can then override them. If I'd allow you to just change all the files, then that would be impossible to update.

Almost any plugin's options can be changed with `opts=`. Just refer to the docs for the plugin or to https://www.lazyvim.org/ if you want to see the defaults that were set.

You should not copy over all the keymaps. That makes zero sense. Read the docs on General Settings.
